test/testdata/config2

```diff
@@ -14,7 +14,7 @@ contexts:
     cluster: cluster1
     user: user2
   name: user2@cluster1
-current-context: ""
+current-context: "user1@cluster1"
```

Comment:
Please no modifications to this file. It might impact other tests. You can set the value in the test.